mathics/builtin/image/colors.py

```diff
@@ -41,77 +41,112 @@ class Binarize(Builtin):
       <dd>map $t1$ < $x$ < $t2$ to 1, and all other values to 0.
     </dl>
 
-    S> hedy = Import["ExampleData/hedy.tif"];
-    S> Binarize[hedy]
+    >> hedy = Import["ExampleData/hedy.tif"];
```

Comment:
We no longer use temporary files. So this should be fine for sand boxing. 

@suhr - would you double check?